build-logic-commons/build.gradle.kts

```diff
@@ -1,3 +1,14 @@
+subprojects {
```

Comment:
Let's not use the `subprojects` block, since it is not compatible with isolated projects. This should go in a convention plugin instead. I suppose we'd need to have a `build-logic-conventions` included build which sets the general conventions for the build logic builds. Though then that build also needs a build file. That would mean that we need to repeat the logic at least once.

For now, I'd prefer the repetition over using `subprojects`.